src/Microsoft.PowerApps.TestEngine/PowerApps/PowerAppFunctions.cs

```diff
@@ -194,8 +194,6 @@ private async Task CheckAndHandleIfLegacyPlayerAsync()
 
         public async Task<Dictionary<string, ControlRecordValue>> LoadPowerAppsObjectModelAsync()
         {
-            await CheckAndHandleIfLegacyPlayerAsync();
-
             await PollingHelper.PollAsync<bool>(false, (x) => !x, () => CheckIfAppIsIdleAsync(), _testState.GetTestSettings().Timeout, _singleTestInstanceState.GetLogger(), GetAppStatusErrorMessage);
```

Comment:
Just a thought, might be a good idea to decouple this too? This method checks for appstatus and LoadPowerAppsObjectModelAsync() seems to do more than one work (appstatus and buildobjectmodel). Both async calls